iowa-a/bedrock-prod/clock-deploy.yaml

```diff
@@ -63,6 +63,23 @@ spec:
               key: CLUSTER_NAME
         - name: CONTENT_CARDS_BRANCH
           value: prod-processed
+        - name: CONTENTFUL_NOTIFICATION_QUEUE_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              key: contentful-notification-access-key-id
+              name: bedrock-prod-secrets
+        - name: CONTENTFUL_NOTIFICATION_QUEUE_REGION
+          value: us-west-2
+        - name: CONTENTFUL_NOTIFICATION_QUEUE_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              key: contentful-notification-secret-access-key
+              name: bedrock-prod-secrets
+        - name: CONTENTFUL_NOTIFICATION_QUEUE_URL
+          valueFrom:
+            secretKeyRef:
+              key: contentful-notification-queue-url
+              name: bedrock-prod-secrets
```

Comment:
Compare all this with https://github.com/mozmeao/www-config/blob/master/iowa-a/bedrock-dev/clock-deploy.yaml#L59